homeassistant/components/panel_iframe/__init__.py

```diff
@@ -46,7 +48,7 @@ async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
             info.get(CONF_TITLE),
             info.get(CONF_ICON),
             url_path,
-            {"url": info[CONF_URL]},
+            {CONF_URL: info[CONF_URL], CONF_ALLOW: info[CONF_ALLOW]},
```

Comment:
Integration tests need to be updated

Reply:
@Quentame Had a stab at a test in https://github.com/home-assistant/core/pull/108230/commits/ac6da572b5c4ed4233e2696c6a782ed13ddb5132. Hope this works?!